presto-docs/src/main/sphinx/functions/array.rst

```diff
@@ -21,6 +21,20 @@ The ``||`` operator is used to concatenate an array with an array or an element
 Array Functions
 ---------------
 
+.. function:: all_match(array(T), function(T,boolean)) -> boolean
+
+    Returns whether all elements of an array match the given predicate. Returns ``true`` if all the elements
+    match the predicate (a special case is when the array is empty); ``false`` if one or more elements don't
+    match; ``NULL`` if the predicate function returns ``NULL`` for one or more elements and ``true`` for all
+    other elements.
```

Comment:
What about empty array? Every predicate would return true? Is that worth documenting?

Reply:
@rongrong : It's already discussed:
```
(a special case is when the array is empty)
```